.github/workflows/triage.yml

```
@@ -7,22 +7,23 @@ concurrency:
   cancel-in-progress: true
 
 permissions:
-  contents: read
+  pull-requests: write
 
 jobs:
   triage:
     runs-on: ubuntu-22.04
     steps:
       - name: Check commit format
+        if: ${{!contains(github.event.pull_request.labels.*.name, 'CI-published-bottle-commits')}}
```

Maybe we can simplify it just a little bit:

```suggestion
        if: !contains(github.event.pull_request.labels.*.name, 'CI-published-bottle-commits')
```

`${{ }}` wouldn't hurt, though; we use both styles in our workflows. But line 24 is already using `if: always()`.

---

Yea, but dropping the `${{ }}` ruined the syntax highlighting in my editor... 😅 It didn't like the `!`.

---

Also, I clicked merge just as your comment came in! I didn't mean to ignore it.

---

Ah, never mind then!

---

No worries! Yep, I noticed that they happened almost simultaneously.